circleci/images/pgupgradetester/files/sbin/install-testdeps

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# make bash behave
+set -euxo pipefail
+IFS=$'\n\t'
+
+# this script needs root to install packages
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root"
+   exit 1
+fi
+
+OLD_PG_VERSION=10
```

Comment:
I think here we can rename this and have something like `pg_versions=( 10 11 )`, in the near future we will have pg 12 support as well and at the beginning we wont drop pg 10 support. So in that case this will be updated as `pg_versions=( 10 11 12 )`.  What do you think? Do you want to replace them to Dockerfile so that every configuration is passed from there?

Reply:
given we don't need to switch postgres versions often I am ok with having it in code.
I like the idea of not limiting to 2 versions but use a list of versions to bundle, the image is slightly bigger but probably not a huge problem. We do the same for the `extbuilder` I believe.